app/modules/listings/listings.js

```diff
@@ -1,31 +1,21 @@
 import app from '#app/app'
-import { getListingMetadata, getListingPathname, getListingWithElementsById } from './lib/listings.js'
 
 export default {
   initialize () {
     const Router = Marionette.AppRouter.extend({
       appRoutes: {
-        'lists/(:id)(/)': 'showListing',
```

Comment:
dont we lose access to a list by deleting this line ?

Reply:
Indeed, that was a mistake. Fixed in the amended commit bac809abe